kyverno/authorization/protect-default-scc/protect-default-scc.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: protect-default-scc
 spec:
   background: true
-  validationFailureAction: enforce
+  validationFailureAction: audit
```

Comment:
@michaelkotelnikov  This message is received because the SCC 'node-exporter' has been modified after the policy is applied. The ClusterpolicyReport is logging this violation, if it was enforced the scc modification was denied.

Reply:
As long as it's not consistent, I'm ok with approving this PR.